backends/dart_minerva/lib/builders/loggers_builder.dart

```diff
@@ -0,0 +1,13 @@
+import 'package:minerva/minerva.dart';
+
+class LoggersBuilder extends MinervaLoggersBuilder {
```

Comment:
I'm not sure how it works but I wanted to confirm that won't log while running in production for these benchmarks

Reply:
This is just the standard code for connecting loggers to the framework. In this example, it will only output a message that the server is running. This will not affect the test in any way later.

Reply:
Then it is fine 👍